lib/pages/settings_dashboard/settings_profile/settings_profile.dart

```diff
@@ -215,7 +228,7 @@ class SettingsProfileController extends State<SettingsProfile>
     return imagePickerController;
   }
 
-  void setAvatarAction() async {
+  void onTapAvatarInMobile() async {
     final action = actions().isEmpty
         ? actions().single.key
```

Comment:
why actions is empty but still have single ?

Reply:
My fault, I'll remove it